apple/Handlers/RNHoverHandler.m

```diff
@@ -167,13 +168,51 @@ @implementation RNHoverGestureHandler
 
 - (instancetype)initWithTag:(NSNumber *)tag
 {
-  RCTLogWarn(@"HoverGestureHandler is not supported on macOS");
   if ((self = [super initWithTag:tag])) {
     _recognizer = [NSGestureRecognizer alloc];
   }
+
   return self;
 }
 
+- (void)bindToView:(RNGHUIView *)view
+{
+  for (NSTrackingArea *trackingArea in view.trackingAreas) {
+    [view removeTrackingArea:trackingArea];
+  }
+
+  NSTrackingAreaOptions options =
+      NSTrackingMouseEnteredAndExited | NSTrackingActiveInActiveApp | NSTrackingInVisibleRect;
```

Comment:
Here we can use `NSTrackingActiveAlways` instead of `NSTrackingActiveInActiveApp` - it will result in `Hover` being active even if window is not focused. 

I'm not sure if we want this behavior though (and making it customizable only on `macOS` doesn't make much sense to me). Let me know what you think @j-piasecki

Reply:
I'd like to keep it consistent - if it always works on web on macOS then we can do that. If web requires a window to be focused then we can keep it this way.

Reply:
I've just checked that and it requires window to be active 😅